### ISSUE TITLE
PlugInstall not working

### ISSUE THREAD

**Issue by cristiAndreiTarasi** [junegunn/vim-plug#741]
Hi
I just switched on Vim-Plug, and after I followed all the steps and typed :PlugInstall in Neovim in windows10 command prompt , instead of installing the plugins threw me out to the cmd. If it helps this is the path to install the plugins - call plug#begin('~/.local/share/nvim/plugged') -. If there is anyone who knows how to fix this, I would apreciate your help. Thank you.
------------------------------
![untitled](https://user-images.githubusercontent.com/18633742/36952168-0d79267c-2004-11e8-9947-db59305d0814.png)

```

```

- Type:

    - [ ] Bug
    - [ ] Enhancement
    - [ ] Feature Request
    -  Question
- OS:
    - [ ] All/Other
    - [ ] Linux
    - [ ] OS X
    - Windows
- Vim:
    - [ ] Terminal Vim
    - [ ] GVim
    - Neovim

**Comment by mschilli87** [comment]
`~/.local/share/nvim/plugged` looks like a Linux or OS X path while you seem to be on Windows.
Could you write down what you did? 'I followed all the steps' doesn't really enable others to reproduce what you did on their own systems. :wink:

**Comment by cristiAndreiTarasi** [comment]
Indeed for windows I should have used an underscore instead of a dot in the path. It still does not work.
I should have mentioned the steps. I followed the steps specified on https://github.com/junegunn/vim-plug for windows.

1. I ran this commands in PowerShell :
`md ~\AppData\Local\nvim\autoload`
`$uri = 'https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'`
`(New-Object Net.WebClient).DownloadFile(`
  `$uri,`
  `$ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath(`
    `"~\AppData\Local\nvim\autoload\plug.vim"`
 ` )`
`) `

2.  Then I placed this in the init.vim : 
`call plug#begin('~/_local/share/nvim/plugged')`
`Plug 'scrooloose/nerdtree'`
`call plug#end()`

3. Then I typed `:PlugInstall` and I've gotten the result in the image. 

I should mention that I repeated the same steps in Vim  8.0.1476
 and it worked. I just want to make it work with Neovim.
